pkg/render/apiserver.go

```diff
@@ -499,6 +499,14 @@ func allowTigeraAPIServerPolicy(cfg *APIServerConfiguration) *v3.NetworkPolicy {
 		},
 	}...)
 
+	if r, e := cfg.K8SServiceEndpoint.DestinationEntityRule(); r != nil && e == nil {
```

Comment:
can we just call the error `err`? Made me do a double-take here and I don't think there's a code reason it needs to be named something else.